src/index.js

```diff
@@ -0,0 +1,96 @@
+// change the scene depending on the sky
+// make a reset city name button
+// add a "landscape" element
+
+const state = {
+  clickCount: 50,
+};
+
+const colors = (clicks, text) => {
+  if (clicks < 50) {
+    text.style.color = 'blue';
+  } else if (clicks < 59) {
+    text.style.color = 'green';
+  } else if (clicks < 69) {
+    text.style.color = 'pink';
+  } else if (clicks < 79) {
+    text.style.color = 'orange';
+  } else {
+    text.style.color = 'red';
+  }
+};
+
+const landscape = (clicks, container) => {
+  if (clicks < 50) {
```

Comment:
The checks that are being done for the colors and landscape are very similar. What if we packed the values we wanted to set into a data structure that combined the effective temperature range, the desired color, and the landscape string? Then we could loop over the values, checking for the range that the temperature is in, and set the appropriate value. 

Something like
```js
  const tempRecords = [
    { upperBound: 50, color: "blue", landscape: "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲"},
    { upperBound: 59, color: "other color", landscape: "other landscape"},
    ...
  ];
```

How could we pick the right value to use based on the temperature value?